### DISCUSSION TITLE
IncludedReferenceProduct

### DISCUSSION THREAD

**Discussion by PeterSawatzki-MGGM** [stephanstapel/ZUGFeRD-csharp#285] [Q&A]
In SpecifiedTradeProduct there is no reading/writing of IncludedReferencedProduct. Is this planned ?

**Comment by stephanstapel** [comment]
If you need this functionality, I'd be more than happy if you could contribute the code to the project.
Remember to add reader and writer in all versions where this node is defined along with test cases for both reading and writing.

**Comment by PeterSawatzki-MGGM** [comment]
o.k. will do.

**Reply by HenriSHS** [reply]
I added it with #404 if still needed
